src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsidentifyremandperiodsapi/prisonapi/model/PrisonApiImprisonmentStatus.kt

```diff
@@ -0,0 +1,27 @@
+package uk.gov.justice.digital.hmpps.hmppsidentifyremandperiodsapi.prisonapi.model
+
+import uk.gov.justice.digital.hmpps.hmppsidentifyremandperiodsapi.relevantremand.model.ImprisonmentStatusType
+import java.time.LocalDate
+
+data class PrisonApiImprisonmentStatus(
+  val status: String,
+  val effectiveDate: LocalDate,
+) {
+
+  fun toStatusType(): ImprisonmentStatusType? {
+    if (REMAND_TYPES.contains(this.status)) {
+      return ImprisonmentStatusType.REMANDED
+    } else if (SENTENCE_TYPES.contains(this.status)) {
+      return ImprisonmentStatusType.SENTENCED
+    } else if (RECALL_TYPES.contains(this.status)) {
+      return ImprisonmentStatusType.RECALLED
+    }
+    return null
+  }
+
+  companion object {
+    val REMAND_TYPES = listOf("SEC38")
```

Comment:
this doesn't particular make much of a difference as its a small list but these could be set's rather than list's to make the toStatusType quicker, not sure how many ImprisonmentStatuses a prisoner can have in their life?